### DISCUSSION TITLE
Bring back next export command

### DISCUSSION THREAD

**Discussion by meds** [vercel/next.js#57934] [Ideas]
### Goals

I'd like to be able to run commands to export nextjs and do dev without needing to modify next.config.js

### Background

I build a capacitor mobile app using nextjs which requires the export command, something like this:

`"build:android:prod": "NODE_ENV=production SKIP_SSR=1 next build && next export && npx cap copy android",`

Because of this change now instead of simply using next export in my build:android/build:ios command now I have to comment in/out output: export in next.config.js

### Proposal

Restore next export command so I can build exports from package.json

**Comment by icyJoseph** [comment]
You can define your own `export` script, which could do something like `SHOULD_EXPORT=true next build`

and then inside `next.config.js`

```
// other stuff
output: process.env.SHOULD_EXPORT === 'true' ? "export" : undefined,
// more stuff
```
